DOC-1955: changelog.adoc (#2674)
tinymce/tinymce-docs: modules/ROOT/pages/changelog.adoc

NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].

== 6.4.2 - 2023-04-26

=== Fixed

> * Selection was not correctly scrolled horizontally into view when using the selection.scrollIntoView API.

Reviewer: Other APIs seems to have back ticks treatment in the changelog.

```suggestion
* Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
```

* Menu buttons would have the Tabstopping behaviour in toolbar.
* The editor would display a notification error when it fails to retirieve a blob image uri.

> * Setting an invalid unit in the fontsizeinput would change it do the default value instead of reverting it back to the previous valid value.

Reviewer: Maybe add backticks:

```suggestion
* Setting an invalid unit in the `fontsizeinput` would change it do the default value instead of reverting it back to the previous valid value.
```

> * The urlinput dialog component would not open the typeahead dropdown when the input value was reset to an empty string.

Reviewer: Not sure if this should be wrapped in backticks since it's a type not a real word?

```suggestion
* The `urlinput` dialog component would not open the typeahead dropdown when the input value was reset to an empty string.
```

NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].

== 6.4.2 - 2023-04-26

=== Fixed

* Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.

> * Tab navigation no longer incorrectly stops at menu buttons within toolbar groups

Reviewer 2: ```suggestion
* Tab navigation no longer incorrectly stops at menu buttons within toolbar groups.
```
Missing full-stop no longer missing

> * Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
> * Menu buttons would have the Tabstopping behaviour in toolbar.
> * The editor would display a notification error when it fails to retirieve a blob image uri.
> * Setting an invalid unit in the `fontsizeinput` would change it do the default value instead of reverting it back to the previous valid value.
> * The `urlinput` dialog component would not open the typeahead dropdown when the input value was reset to an empty string.
> * When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on firefox.
> * Context toolbars displayed the incorrect status for the advlist plugin buttons.
> * The image would not be inserted when using the `quickimage` button on Chrome.

Reviewer 2: ```suggestion
* Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
* Tab navigation no longer incorrectly stops at menu buttons within toolbar groups
* The editor displayed a notification error when it failed to retrieve a blob image uri.
* Setting an invalid unit in the `fontsizeinput` changed it to the default value instead of reverting it to the previous, and valid, value.
* The `urlinput` dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.
* When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on Firefox.
* Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
* The image would not be inserted when the `quickimage` button was used on Chrome.
```

Multiple copy edits, and one re-write for meaning.

> === Fixed
> 
> * Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
> * Tab navigation no longer incorrectly stops at menu buttons within toolbar groups.
> * The editor displayed a notification error when it failed to retrieve a blob image uri.
> * Setting an invalid unit in the `fontsizeinput` changed it to the default value instead of reverting it to the previous, and valid, value.
> * The `urlinput` dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.
> * When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on Firefox.
> * Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
> * The image would not be inserted when the `quickimage` button was used on Chrome.

Reviewer 2: ```suggestion
=== Fixed

* The editor displayed a notification error when it failed to retrieve a blob image uri.
* Tab navigation no longer incorrectly stops at menu buttons within toolbar groups.
* The `urlinput` dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.
* Setting an invalid unit in the `fontsizeinput` changed it to the default value instead of reverting it to the previous, and valid, value.
* Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
* Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
* The image would not be inserted when the `quickimage` button was used on Chrome.
* When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on Firefox.
```
Entry order now matches that in TinyMCE’s changelog file, `/github/tinymce/modules/tinymce/CHANGELOG.md`.

> * The editor displayed a notification error when it failed to retrieve a blob image uri.
> * Tab navigation no longer incorrectly stops at menu buttons within toolbar groups.
> * The `urlinput` dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.
> * Setting an invalid unit in the `fontsizeinput` changed it to the default value instead of reverting it to the previous, and valid, value.
> * Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
> * Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
> * The image would not be inserted when the `quickimage` button was used on Chrome.
> * When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on Firefox.

Reviewer 2: ```suggestion
* The editor displayed a notification error when it failed to retrieve a blob image uri.
* Tab navigation no longer incorrectly stops at menu buttons within toolbar groups.
* The `urlinput` dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.
* Redial would, in some situations, cause select elements not to have an initial value selected when they should have.
* Fixed the mouse pointer style from a text cursor to a default arrow pointer when hovering over the tree dialog component items.
* Enabled variant of toggleable `tox-button` and `tox-button--secondary`: it now supports `hover`/`active`/`focus`/`disabled` states.
* Setting an invalid unit in the `fontsizeinput` changed it to the default value instead of reverting it to the previous, and valid, value.
* Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
* Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
* The image would not be inserted when the `quickimage` button was used on Chrome.
```
More edits to bring this into sync with TinyMCE’s changelog, `changelog.md`.

== 6.4.2 - 2023-04-26

=== Fixed

* The editor displayed a notification error when it failed to retrieve a blob image uri.
* Tab navigation no longer incorrectly stops at menu buttons within toolbar groups.
* The `urlinput` dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.
* Redial would, in some situations, cause select elements not to have an initial value selected when they should have.
* Fixed the mouse pointer style from a text cursor to a default arrow pointer when hovering over the tree dialog component items.
* Enabled variant of toggleable `tox-button` and `tox-button--secondary`: it now supports `hover`/`active`/`focus`/`disabled` states.
* Setting an invalid unit in the `fontsizeinput` changed it to the default value instead of reverting it to the previous, and valid, value.
* Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
* Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
* The image would not be inserted when the `quickimage` button was used on Chrome.

> == 6.4.1 - 2023-03-29

Reviewer 3: ![visualstudio-keyboard](https://user-images.githubusercontent.com/64346958/235009234-4d6c6f83-2679-408c-930e-37fb2fcac0fa.png)
[README.md](https://github.com/tinymce/tinymce-docs/files/11348552/README.md)
